frontend/public/locales/en/translation.json

```diff
@@ -3,5 +3,22 @@
     "description": {
         "part1": "To get started, edit <1>src/App.js</1> and save to reload.",
         "part2": "Switch language between english and german using buttons above."
+    },
+    "users": {
```

Comment:
As I read, it's better to put all translations in `"translation"` object and then split them into categories

Reply:
Not sure that I understood what you mean. Do you mean putting `Dialog` into the root of this json?